src/main/java/se/sundsvall/messaging/integration/digitalmailsender/DigitalMailSenderClient.java

```diff
@@ -16,4 +18,7 @@ interface DigitalMailSenderClient {
 
     @PostMapping("/sendDigitalMail")
     ResponseEntity<DigitalMailResponse> sendDigitalMail(DigitalMailRequest request);
+
+    @PostMapping("/sendDigitalInvoice")
```

Comment:
This comment applies to DigitalMailSender, but shouldn't this path be in lower-case?

https://dev.dataportal.se/rest-api-profil/url-format-och-namngivning

Reply:
As stated: out of scope, but may very well be a candidate for change with everything else cleanup-related going on in DMS

Reply:
Done in https://github.com/Sundsvallskommun/api-service-messaging/pull/107